src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -259,7 +259,7 @@ def load_arguments(self, _):
         c.argument('target_folder', options_list=['--target-folder'], help='Destination folder to which content will be restored. To restore content to root , leave the folder name empty')
         c.argument('target_storage_account', options_list=['--target-storage-account'], help='Destination storage account to which content will be restored')
         c.argument('source_file_type', arg_type=get_enum_type(['File', 'Directory']), options_list=['--source-file-type'], help='Specify the source file type to be selected')
```

Comment:
>source_file_type [](start = 20, length = 16)

all the file must be the same type? or you should implement command like --source-file type='json' path="/e"  --source-file type='txt' path="/b"

Reply:
also the target_folder

---
In reply to: [390722472](https://github.com/Azure/azure-cli/pull/12522#discussion_r390722472) [](ancestors = 390722472)

Reply:
Yes, we are allowing restore of multiple items of same type i.e. file or directory. Point to be mentioned here is source_file_type is not the extension of the file. It simply specifies if the path given by the user is a file or a directory